versioned_docs/version-V3/guides/liquidity-mining/overview.md

```diff
@@ -0,0 +1,42 @@
+---
+id: liquidity-mining-overview
+title: Overview
+sidebar_position: 1
+---
+
+## Introduction
+
+As a DeFi project, token creator, or other interested party, you may be wondering what the best ways to _incentivize in-range liquidity provision_ on one or more Uniswap V3 pools are. This guide describes one particular incentivization scheme at a high level, as implemented in [uniswap-v3-staker](https://github.com/Uniswap/uniswap-v3-staker).
+
+## The Setting
+
+Let's start by defining some terms. First, we refer to programs which incentivize liquidity as `Incentive`s. `Incentive`s are characterized by the following parameters:
+
+- `rewardToken`: Perhaps the most important parameter, would-be incentivizers must pick the ERC20 token which they would like to distribute as a reward for providing active liquidity.
+- `pool`: The address of the Uniswap V3 pool in which liquidity must be provided.
+- `startTime`: The UNIX timestamp at which rewards start to be distributed.
+- `endTime`: The UNIX timestamp at which rewards start to decay.
+- `refundee`: The address which has the right to reclaim any leftover rewards after the `Incentive` has concluded.
+
+## Staking
+
+So, now that we have an idea of what an `Incentive` looks like, let's try understand how a user might participate. Note that this section requires a basic understanding of how Uniswap V3 position NFTs work.
```

Comment:
I expect this is on your todo list, but chronologically should we first explain how to create a program?

Reply:
do you mean operationally? that's not quite how i was envisioning this guide being used, might make sense to have a much more brief/utilitarian page about actually creating an incentive. also, that should probably reference the multi-incentive wrapper contract we're going to write, rather than the staker directly (TKTK)

Reply:
Yeah even operationally! If the goal is to encourage devs to create LM programs,  I imagined this guide as a walkthrough of the process to create, distribute, refund. Happy to put it in a separate guide

Reply:
reworked the order to be more chronological, other point still stands